src/main/java/sixgaezzang/sidepeek/users/controller/UserController.java

```diff
@@ -15,12 +20,20 @@
 
 @RestController
 @RequestMapping("/users")
```

Comment:
기존에 저희가 정한 
```suggestion
@RequestMapping("/api/v1/users")
```
[여기에](https://github.com/side-peek/sidepeek_backend/pull/39#discussion_r1490027403)도 적어놓긴 했는데..! 

기존 노션의 API 명세서와 URI가 달라서 확인 한 번만 부탁드립니다!